### PR TITLE
Update brew install for Linux

### PR DESCRIPTION
**Pull request by rothgar**
Homebrew instructions work on Linux too.

Resolves #

## Target Release

1.6.0

### CODE REVIEW

**Comment by janosdebugs** [comment]
Hey @rothgar thank you very much for your contribution. It seems that we would need to change this file and rename it too: https://github.com/opentofu/opentofu/blob/main/website/docs/intro/install/macos.mdx

**Comment by rothgar** [comment]
Thanks for the tip on the file I missed. I renamed that file to homebrew but I couldn't find instructions on how to run the site locally to check if it shows up properly in the sidebar.

File: website/docs/intro/install/index.mdx [hunk 1]
```diff
@@ -43,9 +43,9 @@ You can install OpenTofu via a wide range of methods. Please select your operati
         {
             type: "link",
             href: "/docs/intro/install/macos",
```
Comment (janosdebugs) [inline website/docs/intro/install/index.mdx:45]: I believe this should be changed too. You can test the website using the new container setup in the [website repo](https://github.com/opentofu/opentofu.org) to make sure everything works.

**Review by janosdebugs — DISMISSED** [review]

**Comment by rothgar** [comment]
I think that did it. @janosdebugs let me know if you need something else

**Comment by Yantrio** [comment]
Hi @rothgar , all 3 of your commits need to be signed off or squashed for the DCO check to pass. Sorry about this!

**Review by cam72cam — APPROVED** [review]

**Pull request merged**